scripts/rewrite/verify-ticket.sh

```
@@ -0,0 +1,460 @@
+  log "  (no test files to run — scaffold-only ticket)"
+  record "unit-tests-run" "pass" "no tests to run (scaffold)"
+else
+  if [ ! -d "$REPO_ROOT/backend_v2" ]; then
+    fail "backend_v2/ directory does not exist yet — cannot run unit tests"
+    record "unit-tests-run" "fail" "backend_v2/ missing"
+    post_pr_summary "FAIL (unit tests — no backend_v2/)"
+    exit 3
+  fi
+  log "  running: uv run pytest ${TESTS_TO_RUN} -q ${COV_TARGETS} --cov-fail-under=${COVERAGE_FLOOR}"
+  if ! (cd "$REPO_ROOT/backend_v2" \
+        && uv run pytest $TESTS_TO_RUN -q $COV_TARGETS --cov-fail-under="$COVERAGE_FLOOR" 2>&1); then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)